src/transpiler.js

```diff
@@ -95,6 +97,14 @@ function getFamilyName({ names = undefined }) {
   return names && names[0] && names[0].familyName
 }
 
+function getCompany({ organizations = undefined }) {
+  return organizations && organizations[0].name
+}
+
+function getNote({ userDefined = undefined }) {
+  return (userDefined && userDefined.find(ud => ud.key === 'note').value) || ''
```

Comment:
Samr as in th previous PR, should we keep either `undefined` or `''` as default value?

Reply:
yeah, I just rebased master to fix this, thanks